queries/all_brain_measures.sparql

```diff
@@ -0,0 +1,29 @@
+# This query simply returns the brain measurements without dependencies on other 
+# demographics/assessment measures.
+
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix prov: <http://www.w3.org/ns/prov#>
+prefix ndar: <https://ndar.nih.gov/api/datadictionary/v2/dataelement/>
+prefix nidm: <http://purl.org/nidash/nidm#>
+prefix dct: <http://purl.org/dc/terms/>
+prefix dctypes: <http://purl.org/dc/dcmitype/>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+select distinct ?ID ?tool ?softwareLabel ?federatedLabel ?laterality ?volume
+where {
+        ?tool_act a prov:Activity ;
+                    prov:qualifiedAssociation [prov:agent [nidm:NIDM_0000164 ?tool]] .
+                ?tool_act prov:qualifiedAssociation [prov:agent [ndar:src_subject_id ?ID]] .
+                ?tool_entity prov:wasGeneratedBy ?tool_act ;
+                        ?measure ?volume .
+
+                ?measure a/rdfs:subClassOf* nidm:DataElement ;
+                            rdfs:label ?softwareLabel;
+                            nidm:measureOf <http://uri.interlex.org/base/ilx_0112559> ; #Volume
+                            nidm:datumType <http://uri.interlex.org/base/ilx_0738276> . #Volume
+                            #nidm:measureOf <http://uri.interlex.org/base/ilx_0111689> . #Thickness
+                            #nidm:measureOf <http://purl.obolibrary.org/obo/PATO_0001323> . #Surface Area
+                OPTIONAL {?measure nidm:isAbout ?federatedLabel }.
+                OPTIONAL {?measure nidm:hasLaterality ?laterality }.
+}
+ORDER BY ?ID ?softwareLabel
```

Comment:
would it be sufficient to become consistent across runs?  may be add more into the mix here, e.g.
```suggestion
ORDER BY ?ID ?softwareLabel ?tool @federatedLabel
```
?

Reply:
I was thinking of adding a study name to differentiate the output and aid referencing sourced tools/data, is there such a label?

Reply:
I dunno... I just see above that you are selecting only `select distinct ?ID ?tool ?softwareLabel ?federatedLabel ?laterality ?volume` . IMHO better that to start with, get some samples from the "lake", and see then if things change when others to run the queries

Reply:
I ran it against the ReproLake_dev and it looks a bit cleaner with tools as the second ordering:

ORDER BY ?ID ?tool ?softwareLabel